docker.go

```diff
@@ -961,11 +961,16 @@ func (p *DockerProvider) CreateContainer(ctx context.Context, req ContainerReque
 
 	sessionID := sessionID()
 
+	reaperOpts := containerOptions{}
```

Comment:
Would you want some mechanism to fall back to the deprecated `req.ReaperImage`? Perhaps initalize `reaperOpts` with the deprecated req.ReaperImage?
```suggestion
	reaperOpts := containerOptions{
	   ImageName: req.ReaperImage
	}
```

Reply:
Indeed, makes total sense